qemu/accel/tcg/cputlb.c

```diff
@@ -1436,7 +1438,28 @@ load_helper(CPUArchState *env, target_ulong addr, TCGMemOpIdx oi,
     bool handled;
     HOOK_FOREACH_VAR_DECLARE;
     struct uc_struct *uc = env->uc;
-    MemoryRegion *mr = memory_mapping(uc, addr);
+
+    /* Handle CPU specific unaligned behaviour */
+    if (addr & ((1 << a_bits) - 1)) {
```

Comment:
Any reason to bring this forward?

Reply:
I have split up the definition and the initialization, if you meant something else please explain.